View, create, and use tokens with the API (#3049)
influxdata/docs-v2: content/influxdb/v2.0/api-guide/api_intro.md

### Authentication

InfluxDB uses [API tokens](/influxdb/v2.0/security/tokens/) to authorize API requests.

> 1. Before diving into the API, use the InfluxDB UI to

Reviewer: Less colloquial but still expressive? Just an idea.

```suggestion
1. Before exploring the API, use the InfluxDB UI to
```

Author: Thanks - debated removing it.

[create an initial API token](/influxdb/v2.0/security/tokens/create-token/) for your application.

> 2. Include your API token as an `Authorization: Token yourApiToken` header in each request.

Reviewer: ```suggestion
2. Include your API token with each request as a head with the form: `Authorization: Token yourApiToken`.
```
